### PR TITLE
Updated products section

### PR DESCRIPTION
**Pull request by sushmangupta**
Related to [DX-189](https://shopware.atlassian.net/browse/DX-189)

### CODE REVIEW

File: products/community-edition.md [hunk 1]
```diff
@@ -2,40 +2,40 @@
...
+The Shopware Platform itself is a Symfony application which consists of several components developed as Symfony bundles. Each of these components is available throughout many repositories and is included in the Shopware Platform mono repository. For the time being, there is the Core component, which includes the framework and business logic, as well as the APIs. The [Storefront](../guides/plugins/plugins/storefront/) component is a default frontend for your Storefront built upon the Bootstrap toolkit and Twig templates. A Vue.js SPA [Administration](../concepts/framework/architecture/administration-concept.md) component wrapped inside a Symfony bundle is a default Administration panel for all back-office tasks and communicates via the [Admin API](../concepts/api) with the Core component. Last but not least, the Elasticsearch component gives you the opportunity to improve the indexing of entities and also contains an adapter for the entity search.
```
Comment (elkmod) [inline products/community-edition.md:13]:
```suggestion
The Shopware Platform itself is a Symfony application which consists of several components developed as Symfony bundles. Each of these components is mirrored to a many repository and included in the Shopware Platform mono repository. For the time being, there is the Core component, which includes the framework and business logic, as well as the APIs. The [Storefront](../guides/plugins/plugins/storefront/) component is a default frontend for your Storefront built upon the Bootstrap toolkit and Twig templates. A Vue.js SPA [Administration](../concepts/framework/architecture/administration-concept.md) component wrapped inside a Symfony bundle is a default Administration panel for all back-office tasks and communicates via the [Admin API](../concepts/api) with the Core component. Last but not least, the Elasticsearch component gives you the opportunity to improve the indexing of entities and also contains an adapter for the entity search.
```

Comment (elkmod) [reply products/community-edition.md:13]: It's actually called a "many/multi repository" and a "mono repository" - it's a pattern of organizing bigger software projects. 

https://en.wikipedia.org/wiki/Monorepo

File: products/extensions/b2b-suite.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
-# B2B Suite
```
Comment (elkmod) [inline products/extensions/b2b-suite.md:1]: It's proper name is "B2B Suite"

File: products/extensions/b2b-suite.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
...
+# B2B-Suite
 
-* [B2B Suite](b2b-suite.md)
+* [B2B-Suite](b2b-suite.md)
```
Comment (elkmod) [inline products/extensions/b2b-suite.md:3]: Same here

File: products/extensions/b2b-suite/guides/installation/README.md [hunk 4]
```diff
@@ -36,36 +36,35 @@ As minimum requirement, we need a docker runtime with version 1.12.* or higher.
 ./psh.phar docker:destroy   # clear the whole docker cache
 ```
 
-To start the docker environment just type
+To start the Docker environment, just type in your command line.
 
 ```bash
-./psh.phar docker:start
+./psh.phar Docker:start
```
Comment (elkmod) [inline products/extensions/b2b-suite/guides/installation/README.md:42]: No uppercase commands.

File: products/extensions/b2b-suite/guides/installation/README.md [hunk 5]
```diff
@@ -36,36 +36,35 @@ As minimum requirement, we need a docker runtime with version 1.12.* or higher.
...
 ```
 
-on your command line. The several containers are booted and afterwards you can login into your web container with
+Several containers are booted. Later you can login into your web container with:
 
 ```bash
-./psh.phar docker:ssh
+./psh.phar Docker:ssh
```
Comment (elkmod) [inline products/extensions/b2b-suite/guides/installation/README.md:48]: No uppercase commands.

File: products/extensions/b2b-suite/guides/storefront/acl-routing.md [hunk 6]
```diff
@@ -38,32 +27,32 @@ Shopware\B2B\AclRoute\Framework\AclRoutingUpdateService::create()
...
-| delete         |                            removal of existing entities                             |
-| update         |                         updating/Changing existing entities                         |
-| assign         |                        changing the assignment of the entity                        |
-| free           |                                   no restrictions                                   |
-
-It is quite natural to map CRUD Actions like this. However, assignment is a little less intuitive. This should help:
-
-* All assignment controllers belong to the resource of the right side of the assignment (e.g. `B2BContactRole` controller is part of the `role` resource).
-* All assignment listings have the detail privilege (e.g. `B2BContactRole:indexAction` is part of the `detail` privilege).
+| List           |                  Entity listing (e.g. indexActions, gridActions)                   |
```
Comment (elkmod) [inline products/extensions/b2b-suite/guides/storefront/acl-routing.md:38]: Also here, I suppose the casing of the privilege is important. It should probably even be in code fences.

File: products/extensions/b2b-suite/guides/storefront/company.md [hunk 7]
```diff
@@ -1,32 +1,23 @@
...
 
-The `CompanyFilterStruct` is used by company module to filter and search for the entities. It extends the `SearchStruct` by the `companyFilterType` and `aclGrantContext`. The correct filter type can be applied by the `CompanyFilterHelper`, possible filter types see in the list below.
+The `CompanyFilterStruct` is used by the company module to filter and search for entities. It extends the `SearchStruct` by the `companyFilterType` and `aclGrantContext`. The correct filter type can be applied by the `CompanyFilterHelper`. Possible filter types are in the list below:
 
 | Filter name   |                        What it applies                         |
 |---------------|:--------------------------------------------------------------:|
-| acl           |  shows only entities which are visible to this `grantContext`  |
-| assignment    |      shows only entities assigned to this `grantContext`       |
-| inheritance   |                         shows only entities which are visible to this or inherited `grantContext`s                          |
+| ACL           |  Shows only entities which are visible to this `grantContext`  |
```
Comment (elkmod) [inline products/extensions/b2b-suite/guides/storefront/company.md:21]: Also here, casing is important

File: products/extensions/search/completion.md [hunk 8]
```diff
@@ -1,16 +1,16 @@
...
+The *advanced search* does not use the default [Elasticsearch Completion](https://www.elastic.co/guide/en/elasticsearch/reference/7.10/search-suggesters.html#completion-suggester) because it does only support a fixed order and the storage size is high. As an alternative, *advanced search* uses aggregations to find the most important word combinations for your search input.
 
-The Full Text Boosted field is used to generate a list of completions. Each word is a separate completion suggestion.
+The `FullText` Boosted field is used to generate a list of completions. Each word is a separate completion suggestion.
 
 ## Extension for compound completions
 
 {% hint style="warning" %}
-The default Advanced Search does not support compound completions from multiple words.
+The default *advanced search* does not support compound completions from multiple words.
```
Comment (elkmod) [inline products/extensions/search/completion.md:10]: Advanced Search is the name of the feature, so it should be uppercase everywhere - also the following/previous mentions.

File: products/pwa.md [hunk 9]
```diff
@@ -1,6 +1,6 @@
 # PWA
 
-If you think about providing a decoupled storefront for customers you can use **Shopware PWA** as a base to do that. It is a Single Page Application based on JavaScript and Vue.js, that already integrates a whole lot of the Shopware functionality you know into a very extensible frontend application.
+If you think about providing a decoupled Storefront for customers, you can use **Shopware PWA** as a base. It is a Single Page Application (SPA) based on JavaScript and Vue.js that already integrates a whole lot of the Shopware functionality you know into a very extensible frontend application.
```
Comment (elkmod) [inline products/pwa.md:3]:
```suggestion
If you think about providing a decoupled storefront for customers, you can use **Shopware PWA** as a base. It is a Single Page Application (SPA) based on JavaScript and Vue.js that already integrates a whole lot of the Shopware functionality you know into a very extensible frontend application.
```

Lowercase "storefront" is fine here, as it is not referring to the our component "Storefront", but the general concept of a storefront.

**Review by elkmod — CHANGES_REQUESTED** [review]

File: products/community-edition.md (hunk 1 above)
Comment (sushmangupta) [reply products/community-edition.md]: Okay :)

**Review by elkmod — APPROVED** [review]

**Pull request merged**